run.sh

```diff
@@ -51,6 +51,8 @@ leaderVoteAccountPubkey=$(\
 
 solana-genesis \
   --lamports 1000000000 \
+  --bootstrap-leader-lamports 10000000 \
```

Comment:
Enough lamports for the one node in `./run.sh` to run longer than anybody will ever want it do, until we fix voting (LMK if you want me to!)

Reply:
that node wins all leader scheduling!